apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -162,6 +162,40 @@ export class CaseService {
     return theCase.id
   }
 
+  private addCompletedCaseMessagesToQueue(caseId: string): Promise<void> {
```

Comment:
Why is this function marked as `private` while `addCompletedIndictmentCaseMessagesToQueue` and `addCaseConnectedToCourtCaseMessageToQueue` are not?

Reply:
This one is private because it is only used locally within the case service.